docs/reference/indices/open-close.asciidoc

```diff
@@ -32,3 +32,7 @@ This setting can also be changed via the cluster update settings api.
 
 Closed indices consume a significant amount of disk-space which can cause problems in managed environments. Closing indices can be disabled via the cluster settings
 API by setting `cluster.indices.close.enable` to `false`. The default is `true`.
+
+Closed index responses with `index_closed_exception` and `403` for most APIs, including `_stats` and `_search`. `ignore_unavailable` allows to control whether closed index do not cause `index_closed_exception` to be returned.  Index returns value on call to `_aliases`. 
```

Comment:
I would slightly rephrase this sentence: "Most APIs, including `_stats` and `_search`, return `index_closed_exception` and `403` status code when accessing closed indices. The `ignore_unavailable querystring parameter allows to control whether closed indices cause an error to be returned or not.".